client/starwhale/base/uri.py

```diff
@@ -216,7 +216,11 @@ def real_request_uri(self) -> t.Union[str, Path]:
 
     @property
     def sw_instance_config(self) -> t.Dict[str, t.Any]:
-        return self._sw_config.get_sw_instance_config(self.instance)
+        return (
+            self._sw_config.get_sw_instance_config(self.instance)
+            if not self.instance_alias
```

Comment:
It seems that the `instance_alias` will always have a value.
https://github.com/star-whale/starwhale/blob/5abf68e650c66ebceed81c18f52d48ee8c6f39ed/client/starwhale/base/uri.py#L106-L109

Reply:
done